docker-java-core/src/main/java/com/github/dockerjava/core/util/FiltersBuilder.java

```diff
@@ -64,6 +66,19 @@ public FiltersBuilder withContainers(String... container) {
         return getFilter("container");
     }
 
+    /**
+     * Filter by event types
+     *
+     * @param eventTypes
+     *            array of event types
+     */
+    public FiltersBuilder withEventTypes(String... eventTypes) {
+        withFilter("type",
+            Stream.of(eventTypes)
+                .collect(Collectors.toList()));
```

Comment:
```suggestion
        withFilter("type", Stream.of(eventTypes).collect(Collectors.toList()));
```

Reply:
OK, I rather break things up into multiple lines so people don't have to read horizontally and IntelliJ will also show what type happens in between.